code/modules/projectiles/guns/projectile/pistol/mk58.dm

```diff
@@ -6,7 +6,7 @@
 	item_state = "pistol"
 	origin_tech = list(TECH_COMBAT = 2, TECH_MATERIAL = 2)
 	matter = list(MATERIAL_PLASTEEL = 12, MATERIAL_PLASTIC = 6)
-	price_tag = 1400
+	price_tag = 400
```

Comment:
Did you tested the prices? How heavily it affects the spawn rate in maints? This(and the others) pistol might start appearing literally everywhere with such low prices

Reply:
> Did you tested the prices? How heavily it affects the spawn rate in maints? This(and the others) pistol might start appearing literally everywhere with such low prices

![image](https://user-images.githubusercontent.com/70905548/125954693-1c53702b-61f5-4718-9de3-7b9ee3c82fa7.png)

so probably rarity changes would be required to keep the actual spawn ratios I guess

Reply:
didn't realize that, ill check how often the edited guns spawn